tests/notebooks/with_eval.md

```diff
@@ -2,8 +2,6 @@
 file_format: mystnb
 kernelspec:
   name: python3
-mystnb:
-    execution_mode: 'inline'
```

Comment:
This is specified in the tests. If also specified here, the version here overrides the one specified in the tests, so I removed it here. The tests pass.